circuitpy/lib/farm_ng/utils/packet.py

```diff
@@ -365,61 +365,6 @@ def __str__(self):
         return f"node_state: {self.node_state} ticks_ms: {self.ticks_ms} serial_number: {self.serial_number}"
 
 
-class RmdSpeedCommand(Packet):
```

Comment:
So this is the pattern we want, with packets in the file they're most relevant to? @ethanrublee 
I'm good with it, just want to be sure so I can be consistent going forward.